core/utils/utils_test.go

```diff
@@ -52,30 +47,6 @@ func TestUtils_NewSecret(t *testing.T) {
 	}
 }
 
-func TestUtils_IsEmptyAddress(t *testing.T) {
-	t.Parallel()
-
-	tests := []struct {
-		name string
-		addr common.Address
-		want bool
-	}{
-		{"zero address", common.Address{}, true},
-		{"non-zero address", testutils.NewAddress(), false},
-	}
-
-	for _, test := range tests {
-		test := test
-
-		t.Run(test.name, func(t *testing.T) {
-			t.Parallel()
-
-			actual := utils.IsEmptyAddress(test.addr)
-			assert.Equal(t, test.want, actual)
-		})
-	}
-}
-
 func TestUtils_StringToHex(t *testing.T) {
 	t.Parallel()
 
```

Comment:
Can't add the comment lower. If we do remove BackoffSleeper from `core/utils`, just a note to move the test below too